src/index.js

```diff
@@ -101,13 +101,36 @@ const propertyValueConverters = {
     }
     return propertyValueConverters.backgroundPosition(value)
   },
+  transform(value) {
+    // This was copied and modified from CSSJanus:
+    // https://github.com/cssjanus/cssjanus/blob/4a40f001b1ba35567112d8b8e1d9d95eda4234c3/src/cssjanus.js#L152-L153
+    const nonAsciiPattern = '[^\\u0020-\\u007e]'
+    const unicodePattern = '(?:(?:\\[0-9a-f]{1,6})(?:\\r\\n|\\s)?)'
+    const numPattern = '(?:[0-9]*\\.[0-9]+|[0-9]+)'
+    const unitPattern = '(?:em|ex|px|cm|mm|in|pt|pc|deg|rad|grad|ms|s|hz|khz|%)'
+    const escapePattern = `(?:${unicodePattern}|\\\\[^\\r\\n\\f0-9a-f])`
+    const nmstartPattern = `(?:[_a-z]|${nonAsciiPattern}|${escapePattern})`
+    const nmcharPattern = `(?:[_a-z0-9-]|${nonAsciiPattern}|${escapePattern})`
+    const identPattern = `-?${nmstartPattern}${nmcharPattern}*`
+    const quantPattern = `${numPattern}(?:\\s*${unitPattern}|${identPattern})?`
+    const signedQuantPattern = `((?:-?${quantPattern})|(?:inherit|auto))`
+    const translateXRegExp = new RegExp(`(translateX\\s*\\(\\s*)${signedQuantPattern}(\\s*\\))`, 'gi')
+    const translateRegExp = new RegExp(`(translate\\s*\\(\\s*)${signedQuantPattern}((?:\\s*,\\s*${signedQuantPattern}){0,1}\\s*\\))`, 'gi')
+    const translate3dRegExp = new RegExp(`(translate3d\\s*\\(\\s*)${signedQuantPattern}((?:\\s*,\\s*${signedQuantPattern}){0,2}\\s*\\))`, 'gi')
```

Comment:
@kentcdodds my regex is a bit rusty... I see that you've implemented `backgroundPosition` in a more concise way, what do you suggest for this?

Reply:
Missed your comment. Honestly I rely pretty heavily on the test suite for this project. I'm not super good at regex either 🙃

Reply:
@kentcdodds Gotcha! I'm curious, why are the regex slightly different between this project and cssjanus? Specifically, the huge block of patterns that I copied over don't seem to be used by `backgroundPosition` (there is another simpler pattern). Do you have a preference? Happy to make those changes if they are better / more performant.

Reply:
Yeah, feel free to make updates to improve things so long as the existing tests continue to pass (unless the tests are wrong somehow).

Thanks so much!